src/pykeen/predict.py

```diff
@@ -1007,24 +1007,35 @@ def predict_all(
 
     :return:
         A score pack of parallel triples and scores
+
+    :raises ValueError:
+        if an inductive inference mode is selected, but the model does not support it
     """
     # note: the models' predict method takes care of setting the model to evaluation mode
     logger.warning(
         f"predict is an expensive operation, involving {model.num_entities ** 2 * model.num_real_relations:,} "
         f"score evaluations.",
     )
 
+    if mode is None:
+        num_entities = model.num_entities
+    elif not isinstance(model, InductiveERModel):
+        raise ValueError(f"{mode=} is invalid for a model that does not support inductive inference.")
+    else:
+        inf_num_entities = model._get_entity_len(mode=mode)
```

Comment:
isn't this function also implemented for regular, non-er models, but errors if the mode is anything but None?

Reply:
True, fixed that in c5703c4821e628c1cee013741ce2dfd1aa4899f5